AstroCal/Calendar.py

```diff
@@ -0,0 +1,85 @@
+from kivy.app import App
```

Comment:
calendar.kv and Calendar.py should both be in the view folder. Please may you move them to the view folder and make necessary import changes if need be.

Reply:
I have moved files as requested